src/aliceVision/hdr/DebevecCalibrate.cpp

```diff
@@ -101,10 +101,10 @@ void DebevecCalibrate::process(const std::vector< std::vector< image::Image<imag
       Vec x = solver.solve(b);
       if(solver.info() != Eigen::Success)  return; // solving failed
 
-      std::cout << "system solved" << std::endl;
+//      std::cout << "system solved" << std::endl;
```

Comment:
Use `ALICEVISION_LOG_TRACE("DebevecCalibrate: system solved");`
And same for all other cout.